doc: added release note for map 21.10.1 (#1199)
centreon/centreon-documentation: en/releases/centreon-commercial-extensions.md

## Centreon MAP

### 21.10.1

Release date: `December 16, 2021`

#### Bug fixes

> - Path to map server backup log file in cron is now the proper path

Reviewer: ```suggestion
- Path to map server backup log file in cron is now correct
```

> - Simple link can now be created between a container and a view

Reviewer: ```suggestion
- A simple link can now be created between a container and a view
```

> - Simple link can now be created between a container and a view

Reviewer 2: Through API

- Map server now supports metrics label with '=' and white spaces
- Non-conventional metric labels do not cause map server to crash anymore

> - Configuration file for centreon map server is not replaced during upgrade anymore

Reviewer: ```suggestion
- Configuration file for Centreon map server is not replaced during upgrade anymore
```
